issue-wanted.cabal

```diff
@@ -72,6 +72,8 @@ library
                          -- Core Modules
                          IW.Core.Id
                          IW.Core.Issue
+                         IW.Core.RepoName
+                         IW.Core.RepoOwner
```

Comment:
In general, it's a good practice to create a separate module per data type. But in our case, we have just two small newtypes. You can put them inside `Repo` module directly. We can refactor them later after we start noticing some cyclic dependencies. But there is a big chance we won't have any new data types for a long type. So you will just end up writing 4 lines of imports every time you need to deal with these types and it's just not convenient.